taraxa/state/dpos/contract.go

```diff
@@ -304,7 +337,7 @@ func (self *Contract) upd_staking_balance(beneficiary common.Address, delta *big
 		beneficiary_bal = bigutil.FromBytes(bytes)
 	})
 	was_eligible := beneficiary_bal.Cmp(self.cfg.EligibilityBalanceThreshold) >= 0
-	prev_vote_count := vote_count(beneficiary_bal, self.cfg.EligibilityBalanceThreshold)
+	prev_vote_count := vote_count(beneficiary_bal, self.cfg.EligibilityBalanceThreshold, self.cfg.CoinsPerVote)
```

Comment:
it this ok like this? Shouldn't be applied after hardfork or something? (i dunno current values, just asking)

Reply:
This sets to the same value as eligibitily_threshold after node reboot. So you will have a new field in config, but it will work the same untill values will be changed with a hardfork